wordpress-mobile/WordPress-Android
# Remove `Publish with Confidence` promo dialog and related resources

Fixes #10474 - Removes `Publish with Confidence` dialog + related resource.

To test:
- On a fresh install of the app
- Tap on the New Post button
- Write something and tap Publish
- The `Ready to Publish` dialog should be visible on the screen
![Screenshot 2019-09-03 at 16 12 23](https://user-images.githubusercontent.com/518232/64181000-01946800-ce66-11e9-9d97-a9388281bc76.png)

Tapping `Publish Now` the post is uploaded on the remote server, tapping `Keep Editing` it should not be uploaded.

Update release notes:

- [ x ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.


cc @diegoreymendez - We should remove it from the iOS side as well
